src/broker/cas_cgw.c

```diff
@@ -2438,3 +2456,279 @@ cgw_utype_to_string (int type)
       return (char *) "";
     }
 }
+
+char *
+cgw_rewrite_query (char *src_query)
```

Comment:
```suggestion
cgw_rewrite_query (char *src_query, char *end)
```

Reply:
end means that  "end = strstr (start, REWRITE_DELIMITER_WHERE);"
"end" can be settable before calling this function.

Reply:
fixed it